templates/quiz/start-button.php

```diff
@@ -53,25 +54,51 @@
 						 * @param LLMS_Quiz   $quiz        The current quiz instance.
 						 * @param LLMS_Lesson $lesson      The parent lesson instance.
 						 */
-						echo apply_filters( 'lifterlms_begin_quiz_button_text', __( 'Start Quiz', 'lifterlms' ), $quiz, $lesson );
+						echo apply_filters( 'lifterlms_begin_quiz_button_text', esc_html( $start_button_string ), $quiz, $lesson );
 					?>
 				</button>
+			</form>
+		<?php else : ?>
+			<p><?php esc_html_e( 'You are not able to take this quiz', 'lifterlms' ); ?></p>
+		<?php endif; ?>
+
+		<?php if ( $quiz->is_open() && $quiz->can_be_resumed_by_student() ) : ?>
+			<form method="POST" action="" name="llms_resume_quiz" enctype="multipart/form-data">
 
-			<?php else : ?>
-				<p><?php _e( 'You are not able to take this quiz', 'lifterlms' ); ?></p>
-			<?php endif; ?>
+				<input id="llms-attempt-key" name="llms_attempt_key" type="hidden" value="<?php echo $quiz->get_student_last_attempt_key(); ?>"/>
```

Comment:
This is a good occasion to escape all the output that can be escaped :D can you?